package wise.military.life.util.extension

fun Exception.getErrorMessage() = message ?: "오류를 불러올 수 없어요"
